plugins/usbpro/EnttecUsbProWidget.cpp

```diff
@@ -499,8 +499,8 @@ void EnttecPortImpl::HandleDMXDiff(const uint8_t *data, unsigned int length) {
     uint8_t data[40];
   } widget_data_changed;
 
-  if (length < sizeof(widget_data_changed)) {
-    OLA_WARN << "Change of state packet was too small: " << length;
+  if ((length < 7) || (length > sizeof(widget_data_changed))) {
+    OLA_WARN << "Change of state packet was invalid: " << length;
```

Comment:
Can we change this to "was an invalid size" or similar, we've not actually looked at if the data is valid yet.